### ISSUE TITLE
finish completely removing web folder

### ISSUE THREAD

**Issue by timotheecour** [nim-lang/Nim#9304]
after #9290 and #9233 

* revive #9277

**Comment by kaushalmodi** [comment]
But looks like `nimblepkglist.nim` needs to be kept for now. It seems like the cleanup in done there (until it's deemed that `nimblepkglist.nim` is no longer needed). 

> https://github.com/nim-lang/Nim/pull/9277#issuecomment-428839132

**Comment by timotheecour** [comment]
i thought it was still there as an accident, feel free to close if it's intended to be there

**Comment by timotheecour** [comment]
From IRC (bridge bot) @FromIRC 03:57
<Araq> @narimiran: move nimblepkglist.nim to tools/

**Comment by narimiran** [comment]
PR submitted, let's wait for the tests.
